tests/suites/os/tests/secureboot/index.js

```diff
@@ -1,145 +1,197 @@
 'use strict';
 
-module.exports = {
-	title: 'secure boot tests',
-	tests: [
-		{
-			title: 'check secure boot',
-			run: async function(test) {
-				const securebootEfiVarPath = '/sys/firmware/efi/efivars/SecureBoot-*';
+const securebootEfiVarPath = '/sys/firmware/efi/efivars/SecureBoot-*';
+const Bluebird = require('bluebird');
```

Comment:
This doesn't seem to be used anywhere

Reply:
@alexgg Can you post what error were you getting: https://github.com/balena-os/leviathan/blob/c15d5183a61e4ed9d437642560c2db071b1d7672/core/lib/common/worker.js#L218

I wrote fetchSerial and there is bluebird already imported there?

Reply:
In does seem its not needed - probably some artifact of running it in the middle of refactoring, I'll remove it.